contrib/cryptomb/private_key_providers/source/cryptomb_private_key_provider.cc

```diff
@@ -515,23 +517,26 @@ void CryptoMbPrivateKeyMethodProvider::unregisterPrivateKeyMethod(SSL* ssl) {
 CryptoMbPrivateKeyMethodProvider::CryptoMbPrivateKeyMethodProvider(
     const envoy::extensions::private_key_providers::cryptomb::v3alpha::
         CryptoMbPrivateKeyMethodConfig& conf,
-    Server::Configuration::TransportSocketFactoryContext& factory_context, IppCryptoSharedPtr ipp)
+    Server::Configuration::TransportSocketFactoryContext& factory_context, IppCryptoSharedPtr ipp,
+    std::string& private_key)
     : api_(factory_context.serverFactoryContext().api()),
       tls_(ThreadLocal::TypedSlot<ThreadLocalData>::makeUnique(
           factory_context.serverFactoryContext().threadLocal())),
       stats_(generateCryptoMbStats("cryptomb", factory_context.statsScope())) {
 
   if (!ipp->mbxIsCryptoMbApplicable(0)) {
-    throw EnvoyException("Multi-buffer CPU instructions not available.");
```

Comment:
we should fallback at line 573 also https://github.com/envoyproxy/envoy/pull/29004/files#diff-38f294cd090527c4632a72082be002caf3fdbfca58ea37965190659058032d9fR573

Reply:
ok, done, thanks!